grails-app/views/assessment/index.gsp

```diff
@@ -385,37 +389,20 @@
 						</g:form>
 					</div>
 
-					<%--Dialog box for Assessement Plans--%>
-					<div id="assessment-plan" >
-						<g:form controller="assessmentTechnique" method="post" id="${currentImod.id}" params="[learningObjectiveID: currentLearningObjective.id]">
-							<label>
-								<h2>
-									Learning Objective
-								</h2>
-							</label>
-							<br/>
+<%--Dialog box for Assessement Plans--%>
+<div id="assessment-plan" >
+<g:form controller="assessmentTechnique" method="post" id="${currentImod.id}" params="[learningObjectiveID: currentLearningObjective.id]">

```

Comment:
Each tag should have its own line
``` html
bad
<div><div></div></div>

good
<div>
    <div></div>
</div>
```

And each time there is a tag inside a tag it should be tabbed
``` html
bad
<div>
<div></div>
</div>

good
<div>
    <div></div>
</div>
```